android/build.gradle

```diff
@@ -34,6 +34,15 @@ def getExtOrIntegerDefault(name) {
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
+task extractSo(type: Copy) {
+  from zipTree("${project.rootDir}/../node_modules/react-native/android/com/facebook/react/react-native/0.62.2/react-native-0.62.2.aar")
```

Comment:
Does this line lock us to a specific version of React Native?

Reply:
@jamonholmgren This is the line that lead to me closing the PR for now. This line does tie you to a specific version of react-native and a specific directory structure. It's appropriate for our app but not for upstream.

I'm working now on changes to build.gradle that will not tie you to any specific version of RN. We also need to test this with other versions of react native like 0.60 and 0.61.

Reply:
Sounds good!